src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
Reset styles are supposed to reset the default styles of the HTML elements. 
I saddest you to set font-family in a separated file.